proto/winebridge.proto

```diff
@@ -13,3 +16,32 @@ message MessageRequest {
 message MessageResponse {
     bool success = 1;
 }
+
+message Process {
+    uint32 pid = 1;
+    string name = 2;
+    uint32 threads = 3;
+}
+
+message RunningProcessesRequest {
+}
+
+message RunningProcessesResponse {
+    repeated Process processes = 1;
+}
+
+message CreateProcessRequest {
+    string command = 1;
+    repeated string args = 2;
+}
+
+message CreateProcessResponse {
+    uint32 pid = 1;
+}
+
+message KillProcessRequest {
+    uint32 pid = 1;
+}
+
+message KillProcessResponse {
```

Comment:
If killing the process can fail, we should report the issue here.

Reply:
We already are, as `tonic::State` not as response

Reply:
Right!